build-support/bin/ci.sh

```diff
@@ -298,7 +278,14 @@ if [[ "${run_integration:-false}" == "true" ]]; then
     ./pants.pex --tag='+integration' test.pytest \
       "--test-pytest-test-shard=${python_intg_shard}" \
       src/python:: tests/python:: -- "${PYTEST_PASSTHRU_ARGS[@]}"
-  ) || die "Pants Integration test failure"
+  ) || die "Pants integration test failure"
+  (
+    # NB: We use test_testprojects_integration.py to specifically run the ones that
```

Comment:
This is in reference to `--exclude-target-regexp='.*/testprojects/.*'`?

Reply:
Yes, will clarify.